website/views/static/community-guidelines.jade

```diff
@@ -26,7 +26,7 @@ block content
       .page-header
         h1=env.t('communityGuidelines')
       p.pagemeta
-        |Last updated&nbsp;
+        =env.t('lastUpdated')
```

Comment:
Needs the `&nbsp;` after lastUpdated:

![screen shot 2015-05-03 at 2 35 17 pm](https://cloud.githubusercontent.com/assets/2916945/7446559/d1bf0036-f1a1-11e4-8c93-e9bdf8e27ee8.png)